omnibus/package-scripts/chefdk/postrm

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 cleanup_symlinks() {
-  binaries="chef chef-solo chef-apply chef-shell knife shef ohai berks chef-zero fauxhai foodcritic kitchen rubocop strain strainer chef-client"
```

Comment:
I believe the old one - it's the same as "uninstall old" and then "install new" right?